include/mitsuba/render/blender_types.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+//Blender Mesh format types for the exporter
+namespace blender{
+    static const int ME_SMOOTH = 1 << 0; //smooth shading flag
+
+    //triangle tesselation of the mesh, contains references to 3 MLoop and the "real" face
+    typedef struct MLoopTri {
+		unsigned int tri[3];
+		unsigned int poly;
+    } MLoopTri;
+
+    typedef struct MLoopUV {
+		float uv[2];
+		int flag;
+    } MLoopUV;
+
+    typedef struct MLoopCol {
+		unsigned char r, g, b, a;
+    } MLoopCol;
+
+    typedef struct MLoop {
+		/** Vertex index. */
+		unsigned int v;
+		/** Edge index. */
+		unsigned int e;
+    } MLoop;
+
+    //contains info about the face, like material ID and smooth shading flag
+    typedef struct MPoly {
+		/* offset into loop array and number of loops in the face */
```

Comment:
Super-minor: we changed to ``//`` comments throughout the codebase (for single-line comments).